src/component/dashboard/widget/lastcommented/CommentedAssetList.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import withI18n, {HasI18n} from "../../../hoc/withI18n";
+import {Col, Label, Row, Table} from "reactstrap";
+import TimeAgo from "javascript-time-ago";
+import {injectIntl} from "react-intl";
+import User from "../../../../model/User";
+import {connect} from "react-redux";
+import TermItState from "../../../../model/TermItState";
+import TermIriLink from "../../../term/TermIriLink";
+import RecentlyCommentedAsset from "../../../../model/RecentlyCommentedAsset";
+
+interface CommentedAssetListProps extends HasI18n {
+    locale: string;
+    user: User;
+    commentedAssets: RecentlyCommentedAsset[];
+    loading: boolean;
+}
+
+export const CommentedAssetList: React.FC<CommentedAssetListProps> = props => {
+    const { commentedAssets, loading, i18n, user, locale, formatMessage } = props;
+
+    const renderEmptyInfo = () =>
+       !loading ?
+            <div className="italics py-2">{i18n("dashboard.widget.commentList.empty")}</div> : null;
+
+    const renderMessage = (lastEdited : number, author : User) => {
```

Comment:
Performance-wise, I think it would be more efficient to extract this function out of the component, so that it is not created on every render. Alternatively, `useCallback` could be used.